examples/forms/components/Forms/Label.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import FieldContext from "./FieldContext";
+
+type LabelProps = React.ComponentProps<"label">;
+
+const Label = ({ children }: LabelProps) => {
+  const { id, isRequired } = React.useContext(FieldContext);
```

Comment:
Ja by som toto schoval do `useField`, ale nemusi to byt predmetom tohot PR.

Reply:
môžem spraviť aj v tomto PR. idem na to

Reply:
je to tam